diff --git a/crates/utils/src/core/allocationmanager.rs b/crates/utils/src/core/allocationmanager.rs
index 08f06c3..f46d377 100644
--- a/crates/utils/src/core/allocationmanager.rs
+++ b/crates/utils/src/core/allocationmanager.rs
@@ -19,6 +19,8 @@ library IAllocationManagerTypes {
     clippy::empty_structs_with_brackets
 )]
 pub mod IAllocationManagerTypes {
+    use crate::core::allocationmanager::AllocationManager::OperatorSet;
+
     use super::*;
     use alloy::sol_types as alloy_sol_types;
     /**```solidity
