diff --git a/crates/utils/src/slashing/core/allocationmanager.rs b/crates/utils/src/slashing/core/allocationmanager.rs
index a0a2d66..17a9ead 100644
--- a/crates/utils/src/slashing/core/allocationmanager.rs
+++ b/crates/utils/src/slashing/core/allocationmanager.rs
@@ -19,6 +19,7 @@ library IAllocationManagerTypes {
     clippy::empty_structs_with_brackets
 )]
 pub mod IAllocationManagerTypes {
+    use super::AllocationManager::OperatorSet;
     use super::*;
     use alloy::sol_types as alloy_sol_types;
     #[derive()]
@@ -26,7 +27,7 @@ pub mod IAllocationManagerTypes {
     struct AllocateParams { OperatorSet operatorSet; address[] strategies; uint64[] newMagnitudes; }
     ```*/
     #[allow(non_camel_case_types, non_snake_case, clippy::pub_underscore_fields)]
-    #[derive(Clone)]
+    #[derive(Clone, Hash, PartialEq, Debug, Eq)]
     pub struct AllocateParams {
         #[allow(missing_docs)]
         pub operatorSet: <OperatorSet as alloy::sol_types::SolType>::RustType,
