diff --git i/crates/utils/src/slashing/core/allocationmanager.rs w/crates/utils/src/slashing/core/allocationmanager.rs
index 8fae5b48..8eb9334d 100644
--- i/crates/utils/src/slashing/core/allocationmanager.rs
+++ w/crates/utils/src/slashing/core/allocationmanager.rs
@@ -19,6 +19,8 @@ library IAllocationManagerTypes {
     clippy::empty_structs_with_brackets
 )]
 pub mod IAllocationManagerTypes {
+    use crate::slashing::core::allocationmanager::AllocationManager::OperatorSet;
+
     use super::*;
     use alloy::sol_types as alloy_sol_types;
     /**```solidity
